src/main/java/Entities/Group.java

```diff
@@ -97,5 +97,13 @@ public boolean validateCourseCode(String courseCode){
     public void setGroupName(String name){
         this.groupName = name;
     }
+    public void removeMember(String username) {
+        this.groupMembers.remove(username);
+    }
+    public void removeApplication(String username) {
+        this.memberRequests.remove(username);
+    }
+
+
```

Comment:
Just wondering why these entities are part of this pull request? Have they been changed? Since they are part of mine and Leo's use case i dont think they should be changed in this pull request. Just dont want to have any merge conflicts.